src/routes/api/webhooks/github.ts

```diff
@@ -0,0 +1,300 @@
+          "name": "Apache License 2.0",
+          "spdx_id": "Apache-2.0",
+          "url": "https://api.github.com/licenses/apache-2.0",
+          "node_id": "MDc6TGljZW5zZTI="
+        },
+        "allow_forking": true,
+        "is_template": false,
+        "web_commit_signoff_required": false,
+        "topics": [
+          "aws-amplify",
+          "discord"
+        ],
+        "visibility": "public",
+        "forks": 5,
+        "open_issues": 32,
+        "watchers": 11,
+        "default_branch": "main"
+      },
+      "organization": {
+        "login": "aws-amplify",
+        "id": 41077760,
+        "node_id": "MDEyOk9yZ2FuaXphdGlvbjQxMDc3NzYw",
+        "url": "https://api.github.com/orgs/aws-amplify",
+        "repos_url": "https://api.github.com/orgs/aws-amplify/repos",
+        "events_url": "https://api.github.com/orgs/aws-amplify/events",
+        "hooks_url": "https://api.github.com/orgs/aws-amplify/hooks",
+        "issues_url": "https://api.github.com/orgs/aws-amplify/issues",
+        "members_url": "https://api.github.com/orgs/aws-amplify/members{/member}",
+        "public_members_url": "https://api.github.com/orgs/aws-amplify/public_members{/member}",
+        "avatar_url": "https://avatars.githubusercontent.com/u/41077760?v=4",
+        "description": ""
+      },
+      "enterprise": {
+        "id": 1290,
+        "slug": "amazon",
+        "name": "Amazon",
+        "node_id": "MDEwOkVudGVycHJpc2UxMjkw",
+        "avatar_url": "https://avatars.githubusercontent.com/b/1290?v=4",
+        "description": "",
+        "website_url": "https://www.amazon.com/",
+        "html_url": "https://github.com/enterprises/amazon",
+        "created_at": "2019-11-13T18:05:41Z",
+        "updated_at": "2022-03-18T18:37:08Z"
+      },
+      "sender": {
+        "login": "josefaidt",
+        "id": 5033303,
+        "node_id": "MDQ6VXNlcjUwMzMzMDM=",
+        "avatar_url": "https://avatars.githubusercontent.com/u/5033303?v=4",
+        "gravatar_id": "",
+        "url": "https://api.github.com/users/josefaidt",
+        "html_url": "https://github.com/josefaidt",
+        "followers_url": "https://api.github.com/users/josefaidt/followers",
+        "following_url": "https://api.github.com/users/josefaidt/following{/other_user}",
+        "gists_url": "https://api.github.com/users/josefaidt/gists{/gist_id}",
+        "starred_url": "https://api.github.com/users/josefaidt/starred{/owner}{/repo}",
+        "subscriptions_url": "https://api.github.com/users/josefaidt/subscriptions",
+        "organizations_url": "https://api.github.com/users/josefaidt/orgs",
+        "repos_url": "https://api.github.com/users/josefaidt/repos",
+        "events_url": "https://api.github.com/users/josefaidt/events{/privacy}",
+        "received_events_url": "https://api.github.com/users/josefaidt/received_events",
+        "type": "User",
+        "site_admin": false
+      }
+    },
+  }
+
+  test('Verify webhook', () => {
```

Comment:
Can we add tests for failure cases as well? Like bad/missing signature and error resp from Discord?